lib/zlib.js

```diff
@@ -114,7 +119,7 @@ function zlibBuffer(engine, buffer, callback) {
     var err = null;
 
     if (nread >= kMaxLength) {
-      err = new RangeError(kRangeErrorMessage);
+      err = new errors.RangeError('ERR_INVALID_RANGE', kRangeErrorMessage);
```

Comment:
The second arg here should the be name of the argument, and it needs the range arguments, correct?